lib/tasks/webpacker/clean.rake

```diff
@@ -0,0 +1,15 @@
+require "webpacker/configuration"
+
+namespace :webpacker do
+  desc "Remove old compiled webpacks"
+  task :clean, [:keep] => ["webpacker:verify_install", :environment] do |_, args|
+    Webpacker.clean(Integer(args.keep || 2))
+  end
+end
+
+# Run clean if the assets:clean is run
+if Rake::Task.task_defined?("assets:clean")
+  Rake::Task["assets:clean"].enhance do
+    Rake::Task["webpacker:clean"].invoke
```

Comment:
It would be a very nice touch if we could pass on the same arguments given to `assets:clean`; I'm not sure Rake provides a particularly clean way of doing this though; my best approach so far has been this:

```ruby
Rake::Task["assets:clean"].enhance do |_, args|
  Rake::Task["webpacker:clean"].invoke(*args.to_a)
end
```